.github/workflows/BuildPR.yml

```diff
@@ -53,53 +44,16 @@ jobs:
           MACOS_CERTIFICATE_PWD: ${{ secrets.P12_PASSWORD }}
           MACOS_CERTIFICATE_NAME: ${{ secrets.APPLE_CERTIFICATE_NAME }}
           MACOS_CI_KEYCHAIN_PWD: ${{ secrets.KEYCHAIN_PASSWORD }}
-        run: |
-          # Turn our base64-encoded certificate back to a regular .p12 file
```

Comment:
This pullrequest there is too much stuff, which should be not here.
It touches Windows and MacOS build.
For me, this is mess

I guess the pullrequest should rebased to branch "2.28.x" but doing this, I see a lot of unrelated commits (no feat, I will not push)
<img width="517" height="344" alt="image" src="https://github.com/user-attachments/assets/c57ecfe4-0638-407c-a897-bf6e5db7e73f" />
and I guess these are the only related one 
<img width="522" height="60" alt="image" src="https://github.com/user-attachments/assets/c0219bdb-b8a5-4e1a-a111-1c8264cfa81d" />
am I right ?

I will make a new pull request and keep this untouched